modules/apps/frontend-js/frontend-js-tabs-support-web/src/main/resources/META-INF/resources/TabsProvider.js

```diff
@@ -103,11 +103,15 @@ class TabsProvider {
 		if (activePanels.length) {
 			const activePanel = activePanels[0];
 
-			Liferay.on(this.EVENT_HIDDEN, (event) => {
+			const onHidden = (event) => {
 				if (event.panel === activePanel) {
 					this.show({panel, trigger});
 				}
-			});
+
+				Liferay.detach(this.EVENT_HIDDEN, onHidden);
+			};
+
+			Liferay.on(this.EVENT_HIDDEN, onHidden);
```

Comment:
We can simplify this by using `Liferay.once`

Reply:
@markocikos I updated the pr